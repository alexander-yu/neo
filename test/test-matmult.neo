void main(){
    create matrix<int> a = [[1,2],[3,4],[5,6]];
    create matrix<int> b = [[1,2,3,4,5],[6,7,8,9,10]];
    println(a@b);
    create matrix<float> x = [[1.2,3.5,4.2],[6.2,-1.5,2.3]];
    create matrix<float> y = [[-4.,1.3,5.5],[3.2,2.6,7.2],[1.2,2.3,-3.5]];
    println(x@y);
}
