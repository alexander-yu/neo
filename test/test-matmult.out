[[13, 16, 19, 22, 25],
 [27, 34, 41, 48, 55],
 [41, 52, 63, 74, 85]]
[[11.44, 20.32, 17.1],
 [-26.84, 9.45, 15.25]]
